recipes/sdl_ttf/all/conanfile.py

```diff
@@ -97,6 +117,8 @@ def package_info(self):
         self.cpp_info.includedirs.append(os.path.join("include", "SDL2"))
         self.cpp_info.libs = ["SDL2_ttf"]
         self.cpp_info.requires = ["freetype::freetype", "sdl::libsdl2"]
+        if self.options.get_safe("with_harfbuzz"):
```

Comment:
The package also creates a `SDL2_ttf::SDL_ttf2-static` target when building a static library.

Reply:
I understand.
I will fix it.